JoinColony/colonyCDapp
# Port Step 2 of "Add Safe" dialog

### Testing time!
- Check that all the link works as expected.
- Check that the read-only values make sense.
- Check that the module contract address gets validated successfully both when it's the correct and when it's the wrong address.
- I tested this with these values: `0xe759f388c97674D5B8a60406b254aB59f194163d` (Binance Safe) and `0xCC199aCAb2B38C6DC5f7304B29BDEb3a83E7d212` (bridge module)

![FireShot Capture 116 - Colony CDapp - localhost](https://github.com/JoinColony/colonyCDapp/assets/18473896/911d4505-aa50-4804-aab9-330b2dac56b7)

Resolves #707
